docs/ModelZoo.md

```diff
@@ -0,0 +1,950 @@
+# NNCF Compressed Model Zoo
+
+Here we present the results achieved using our sample scripts, example patches to third-party repositories and NNCF configuration files.
```

Comment:
I suggest to clearly state everywhere if PTQ or QAT was used (assume QAT in all cases now). Perhaps it's better to explain the abbreviations at the beginning and then use them in the tables, it will save some space.

Reply:
Done